### PR TITLE
Add info functions.

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

File: Info/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    name: bytes - utf-8 string with name of debot, e.g. "DePool"
+
+    version: bytes - version of debot, e.g. "0.0.1"
+
+    publisher: bytes - utf-8 string with info about whos deployed debot to blochain, e.g. "TON Labs"
+
+    key: bytes - (10-20 ch.) short description, e.g. "Work with DePools"
+
+    author: bytes - utf-8 string with name of author of DeBot, e.g. "Nikita Monakhov"
```
Comment (Keshoid) [inline Info/README.md]:
```suggestion
    author: bytes - utf-8 string with name of author of DeBot, e.g. "Ivan Ivanov".
```

File: Info/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+`getInfo` - return information about debot
+
+returns:
+
+    name: bytes - utf-8 string with name of debot, e.g. "DePool"
+
+    version: bytes - version of debot, e.g. "0.0.1"
+
+    publisher: bytes - utf-8 string with info about whos deployed debot to blochain, e.g. "TON Labs"
```
Comment (Keshoid) [inline Info/README.md]:
```suggestion
    publisher: bytes - utf-8 string with info about who has deployed debot to blokchain, e.g. "TON Labs"
```

File: MandatoryDeBotFunctions.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+## Description
+
+It's basic functions in debots for getting of info about debot and work with this
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]:
```suggestion
Document describes the functions that every Debot should have.
```

File: MandatoryDeBotFunctions.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+    author: bytes - utf-8 string with name of author of DeBot, e.g. "Ivan Ivanov".
+
+    support: address - Free TON address of author for questions and donations
+
+    hello: bytes - first messsage with DeBot description, e.g. "Hello, I'm a FreeTON Debot Kapellmeister. I know everything about DePools. Let's start." 
+
+---
+
+`getRequiredInterfaces` - returns array of required interfaces.
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]:
```suggestion
`getRequiredInterfaces` - returns array of interfaces used by debot. Debot Browser should check if it supports all interfaces from this array. If anyone is not supported debot should not be started. 
```

File: MandatoryDeBotFunctions.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+returns:
+
+    interfaces: uint256[] - array of interface ids
+
+## Declaration in Solidity
+
+    // Base contract for all debots
+    abstract contract Debot {
+    
+        i32 constant DEBOT_WC = -31;
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]: Remove this line

File: MandatoryDeBotFunctions.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    
+        function getRequiredInterfaces() override returns (uint256[] interfaces) {
+        return [ID_TERMINAL, ID_MENU, ...];
+        }
+    
+    }
+
+## Declaration in C++
+
+TODO: Add declaration
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md:62]: @azhogin Please, add declaration of getRequiredInterfaces() in C++.

File: MandatoryDeBotFunctions.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+## Code Example
+
+### Solidity
+
+TODO: Add example
+
+### C++
+
+TODO: Add example
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]: You can remove lines 56-64

**Review by mikhailUshakoff — APPROVED** [review]

File: MandatoryDeBotFunctions.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+    interfaces: uint256[] - array of interface ids
+
+## Declaration in Solidity
+
+    // Base contract for all debots
+    abstract contract Debot {
+    
+        function getRequiredInterfaces() virtual returns (uint256[] interfaces); 
+
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]:
```suggestion
    function getInfo() virtual returns (string name, string version, string publisher, string key, string author, address support, string hello, string language);
```

File: MandatoryDeBotFunctions.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+        function getRequiredInterfaces() virtual returns (uint256[] interfaces); 
+    
+    }
+    
+    contract DebotA is Debot {
+    
+        function getRequiredInterfaces() override returns (uint256[] interfaces) {
+        return [ID_TERMINAL, ID_MENU, ...];
+        }
+
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]:
```suggestion
    function getInfo() override returns (string name, string version, string publisher, string key, string author, address support, string hello, string language) {
        name = "Depool Debot";
        version = "0.1.0";
        publisher = "TON Labs";
        key = "Works with DePools";
        author = "Ivan Ivanov";
        support = address.makeAddrStd(0, 0x841288ed3b55d9cdafa806807f02a0ae0c169aa5edfe88a789a6482429756a94);
        hello = "Hello, I'm a FreeTON Debot Kapellmeister. I know everything about DePools. Let's start.";
        language = "en_US";
    }
```

File: MandatoryDeBotFunctions.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+        }
+    function getInfo() override returns (string name, string version, string publisher, string key, string author, address support, string hello, string language) {
+        name = "Depool Debot";
+        version = "0.1.0";
+        publisher = "TON Labs";
+        key = "Works with DePools";
+        author = "Ivan Ivanov";
+        support = address.makeAddrStd(0, 0x841288ed3b55d9cdafa806807f02a0ae0c169aa5edfe88a789a6482429756a94);
+        hello = "Hello, I'm a FreeTON Debot Kapellmeister. I know everything about DePools. Let's start.";
+        language = "en_US";
```
Comment (BigRoma) [inline MandatoryDeBotFunctions.md]:
```suggestion
        language = "en";
```

File: MandatoryDeBotFunctions.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+    key: bytes - (10-20 ch.) utf-8 string with short description, e.g. "Work with DePools"
+
+    author: bytes - utf-8 string with name of author of DeBot, e.g. "Ivan Ivanov".
+
+    support: address - Free TON address of author for questions and donations
+
+    hello: bytes - utf-8 string with first messsage with DeBot description, e.g. "Hello, I'm a FreeTON Debot Kapellmeister. I know everything about DePools. Let's start." 
+
+    language: bytes - utf-8 string with debot interface language, e.g. "en_US"
```
Comment (BigRoma) [inline MandatoryDeBotFunctions.md]:
```suggestion
    language: bytes - (ISO-639) utf-8 string with debot interface language, e.g. "en"
```

**Review by Keshoid — APPROVED** [review]

**Pull request merged**